src/main/resources/db/migration/V13__add_requested_status_to_device_updates_table.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `device_updates`
```

Comment:
We're planning to merge this after #103 right? Because in that case the migration should be V12.

Reply:
Yep, that is the idea. I used V13 in order to avoid conflicts later.

Reply:
Actually, after both #103 and #102 because they are ready and this PR is not yet.